test/replication-luatest/commit_wait_test.lua

```diff
@@ -1,58 +0,0 @@
-local t = require('luatest')
```

Comment:
Lets keep the original test file name. The rules are that we use `gh_....` prefix only for bugfixes. For features the test file names don't mention tickets in their title. Usually. Here it seems most appropriate because it covers multiple feature-tickets concerning the same feature - `wait_mode`.

Reply:
Fixed.